src/components/infoCard/InfoCard.tsx

```diff
@@ -0,0 +1,106 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Icon as RNEIcon } from 'react-native-elements';
+
+import { colors, normalize } from '../../config';
+import { Address, Contact, WebUrl } from '../../types';
+import { RegularText } from '../Text';
+import { InfoBox } from '../Wrapper';
+import { AddressSection } from './AddressSection';
+import { ContactSection } from './ContactSection';
+import { UrlSection } from './UrlSection';
+
+type WebUrlProps = {
+  contact?: Contact;
+  contacts?: Contact[];
+  webUrls?: WebUrl[];
+};
+
+type Props = WebUrlProps & {
+  address?: Address;
+  addresses?: Address[];
+  category?: { name?: string };
+  name?: string;
+  openWebScreen: (link: string) => void;
+};
+
+const mergeWebUrls = ({ webUrls, contact, contacts }: WebUrlProps) => {
+  const mergedWebUrls = webUrls ? [...webUrls] : [];
+
+  if (contact?.www) {
+    mergedWebUrls.unshift({ url: contact.www });
+  }
+
+  // merge a `contact`s `webUrls` to `webUrls`
+  if (contact?.webUrls?.length) {
+    mergedWebUrls.push(...contact.webUrls);
+  }
+
+  // iterate all `contacts` and merge every `contact`s `webUrls` to `webUrls`
+  contacts?.forEach((contact) => {
+    if (contact?.www) {
+      mergedWebUrls.push({ url: contact.www });
+    }
+    if (contact?.webUrls?.length) {
+      mergedWebUrls.push(...contact.webUrls);
+    }
+  });
+
+  return mergedWebUrls;
+};
+
+/* TODO: add a logic to display info category and url that fit the screen even if long text
+         (not yet a problem) */
+export const InfoCard = ({
+  address,
+  addresses,
+  category,
+  contact,
+  contacts,
+  name,
+  webUrls,
+  openWebScreen
+}: Props) => (
+  <View>
```

Comment:
this `View` was a `Wrapper` before or am i wrong? does this change rendering visually?

Reply:
but this `{!!name && ( ...` stuff was in operating company before inside of a `<Wrapper>`

<img width="337" alt="Bildschirmfoto 2021-02-19 um 11 21 43" src="https://user-images.githubusercontent.com/1942953/108491798-a8f89b00-72a4-11eb-9523-28ce1e1c908d.png">

Reply:
and it still is in `OperatingCompany` as the new `InfoCard` is wrapped inside a `Wrapper` there.